extensions/mssql/src/tableDesigner/tableDesigner.ts

```diff
@@ -6,21 +6,23 @@
 import { AppContext } from '../appContext';
 import * as azdata from 'azdata';
 import * as vscode from 'vscode';
-import { sqlProviderName, TableType } from '../constants';
+import { sqlProviderName } from '../constants';
 import { generateUuid } from 'vscode-languageclient/lib/utils/uuid';
 import { ITelemetryEventProperties, Telemetry } from '../telemetry';
 
 export function registerTableDesignerCommands(appContext: AppContext) {
 	appContext.extensionContext.subscriptions.push(vscode.commands.registerCommand('mssql.newTable', async (context: azdata.ObjectExplorerContext) => {
 		const connectionString = await azdata.connection.getConnectionString(context.connectionProfile.id, true);
-		const telemetryInfo = await getTelemetryInfo(context, TableType.Basic);
+		const tableSubType = context.nodeInfo.nodeSubType as azdata.designers.TableSubType ?? azdata.designers.TableSubType.Temporal;
+		const telemetryInfo = await getTelemetryInfo(context, tableSubType);
 		await azdata.designers.openTableDesigner(sqlProviderName, {
 			server: context.connectionProfile.serverName,
 			database: context.connectionProfile.databaseName,
 			isNewTable: true,
 			id: generateUuid(),
 			connectionString: connectionString,
-			accessToken: context.connectionProfile.options.azureAccountToken
+			accessToken: context.connectionProfile.options.azureAccountToken,
+			tableSubType: tableSubType
```

Comment:
this is not needed, is it?

Reply:
this is the main entry point for the logic, why wouldn't we need it?

Reply:
we do. you are right.